pkg/testutils/reference.go

```diff
@@ -2,8 +2,8 @@ package testutils
 
 import "github.com/argonsecurity/pipeline-parser/pkg/models"
 
-func CreateFileReference(l1, c1, l2, c2 int) *models.FileReference {
-	return &models.FileReference{
+func CreateFileReference(l1, c1, l2, c2 int, isAlias ...bool) *models.FileReference {
```

Comment:
I know this is bad, but I think the only options are:
1. Add `isAlias` as a `bool` to the function signature (which will obviously change the entire codebase
2. Create another function that receives `isAlias` - I think that this is the better compromise